common/test/acceptance/pages/lms/video/video.py

```diff
@@ -356,9 +356,9 @@ def click_player_button(self, button):
 
         self.q(css=button_selector).first.click()
 
-        button_states = {'play': 'playing', 'pause': 'pause'}
-        if button in button_states:
-            self.wait_for_state(button_states[button])
+        # button_states = {'play': 'playing', 'pause': 'pause'}
+        # if button in button_states:
+        #     self.wait_for_state(button_states[button])
```

Comment:
> Could there be a bug where a class (is-paused) is not getting removed?

No, it [removes all previous](https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/js/src/video/03_video_player.js#L722) states before setting new one.